crates/sui-bridge-indexer/src/storage.rs

```diff
@@ -69,6 +69,13 @@ impl Persistent<ProcessedTxnData> for PgBridgePersistent {
                                         .await?;
                                 }
                             }
+                            ProcessedTxnData::GovernanceAction(a) => {
+                                diesel::insert_into(schema::governance_actions::table)
+                                    .values(&a.to_db())
+                                    .on_conflict_do_nothing()
+                                    .execute(conn)
+                                    .await?;
+                            }
```

Comment:
should we apply the same treatment of eth reorg for this table?

Reply:
I don't think its necessary tbh

Reply:
could you elaborate the rationale?

Reply:
because this is mostly for monitoring, and we don't expect these events to happen that often, I don't think its necessary.

Reply:
based on what we discussed offline, we will be using serial id as primary key and the only unique constraint for this table, which is not ideal but it's what it is. So the original comment becomes a non-issue there will never be any conflicts.